lib/errors.js

```diff
@@ -7,6 +7,12 @@ exports.createTimeoutError = () => {
   return timeoutErr
 }
 
+exports.createRebindError = () => {
+  const reboundErr = new Error('Request socket rebound')
+  reboundErr.code = 'EREBOUND'
+  return reboundErr
+}
+
```

Comment:
For future consideration: Should we use a proper `Error` subclass with error constructor functions for these, same as https://github.com/holepunchto/hypercore-errors?

Reply:
Yea! Can you PR that after I merge?